src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -4621,6 +4620,8 @@ namespace AirflowNetwork {
                 int compnum = compnum_iter->second;
                 AirflowNetworkLinkageData(count).CompNum = compnum;
 
+                auto &surf = m_state.dataSurface->Surface(MultizoneSurfaceData(count).SurfNum);
```

Comment:
Why isn't this `auto const &surf`? I don't see surf on the left side of the =.

Reply:
Because I added this mindlessly because I needed it below.